sky/job/core.py

```diff
@@ -33,20 +33,21 @@ def launch(
     stream_logs: bool = True,
     detach_run: bool = False,
     retry_until_up: bool = False,
-) -> None:
```

Comment:
Keep? Always show -> None rather than omit?

Reply:
Oops, this is a merging error. Fixed. Thanks!